lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Nejde to nějak pomocí <code>pd.DataFrame.from_records</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
records přímo nešlo, ale df jsem z toho udělal

Reply:
<p>No asi můžem nechat, i když dnes jsem ještě přidal další poznámku o from_recods ...</p>